.circleci/config.yml

```diff
@@ -60,6 +60,7 @@ jobs:
           at: .
       - run: scripts/depends.sh
       - run: scripts/build_packages.sh
+      - run: echo ${GOOGLE_AUTH} > ${HOME}/gcp-key.json
```

Comment:
this is not needed.  In scripts/push/gcr.sh you are just loading the contents of this file back into an ENV var.

in scripts/push/gcr.sh  just use ${GOOGLE_AUTH} directly.

Reply:
ok we can do that, as long as we make sure to never add `set -x` in the script. (which i took out now)